locales/en.json

```diff
@@ -1180,8 +1180,8 @@
     "bulk_order": {
       "meta": {
         "title": "Bulk order",
-        "keywords": "test, balk, order, keywords",
-        "description": "Test balk order page description"
+        "keywords": "test, bulk, order, keywords",
+        "description": "Test bulk order page description"
```

Comment:
What's this `test` messages?

Reply:
![image](https://github.com/VirtoCommerce/vc-theme-b2b-vue/assets/19205780/96dcf378-194e-4975-855a-42719d31671e)

Reply:
I think we need to discuss about that with @aavolko and replace this test messages to something production-ready

Reply:
@aavolko remove test